packages/vite/src/node/server/middlewares/indexHtml.ts

```diff
@@ -326,6 +326,6 @@ function preTransformRequest(server: ViteDevServer, url: string, base: string) {
   // transform all url as non-ssr as html includes client-side assets only
   server.transformRequest(url).catch((e) => {
     // Unexpected error, log the issue but avoid an unhandled exception
-    server.config.logger.error(e)
+    server.config.logger.error(e.message)
```

Comment:
Changed this to `e.message` to be consistent with import analysis. I also find this to be sufficient for pre-transform errors instead of logging the whole thing.